### DISCUSSION TITLE
Nextcloud AIO queries DNS server every few seconds

### DISCUSSION THREAD

**Discussion by jlmanohar** [nextcloud/all-in-one#3152] [Questions] [answered]
Any one have any idea why or if its normal that i see lots of DNS requests for my domain that nextcloud is on (cloud.xxxxxxx.org). I am using debian 12, with portainer and have used the example docker compose file to create the stack, i use nginx proxy manager for reverse proxy and followed the setup guide for reverse proxy. I have ipv6 enabled in docker and followed the respective aio guide to setup network and ipv6 tables.

I am using Adguard home a local dns and have written my ip to do a dns rewrite for a, aaaa records. I dont see anything wrong with nextcloud functionality, just too many dns requests ~13-14k per day. All of those come from the docker bridge network.

![image](https://github.com/nextcloud/all-in-one/assets/66161434/20816849-ee55-4d25-adc1-ca933050599d)

![image](https://github.com/nextcloud/all-in-one/assets/66161434/6ca3dea9-c895-422c-8c7d-996717b4646a)

**Comment by jlmanohar** [comment]
Followup information, i see <my domain>.example.org as some of the dns request. Could it be that its something left from test ?
![image](https://github.com/nextcloud/all-in-one/assets/66161434/34a8c5e6-8750-458d-9519-e07aa4915c00)

**Comment by szaimen** [comment, marked as answer]
Hi, most likely the healthcheck is the reason
https://github.com/nextcloud/all-in-one/blob/1771a72c0e5a37d43356c4d3e6c657dc5d4a0990/Containers/apache/healthcheck.sh#L6

**Reply by jlmanohar** [reply]
Thanks, i just added them to the ignore list for my logs and statistics of adguard dns.

**Comment by jlmanohar** [comment]
Any idea on the example.org getting appended on some requests ? i don't see it in the nc_domain env var
![image](https://github.com/nextcloud/all-in-one/assets/66161434/d3af58cf-8030-46a5-b9b9-5a271737b3b7)
